evap/staff/views.py

```diff
@@ -677,48 +678,91 @@ def course_participant_import(request, semester_id, course_id):
     if request.method == "POST":
         operation = request.POST.get('operation')
 
-        if operation not in ('test', 'import', 'copy'):
+        if operation not in ('test-participants', 'import-participants', 'copy-participants',
+                             'test-contributors', 'import-contributors', 'copy-contributors'):
             raise SuspiciousOperation("Invalid POST operation")
 
-        test_run = operation == 'test'
-        import_run = operation == 'import'
-        copy_run = operation == 'copy'
+        if 'participants' in operation:
+            excel_form = participant_excel_form
+            copy_form = participant_copy_form
+        else:                                                 # 'contributors' in operation
+            excel_form = contributor_excel_form
+            copy_form = contributor_copy_form
```

Comment:
if you're creating the magic string with the one-line-if-else, it might make sense to use that here too.